app.js

```diff
@@ -15,20 +15,18 @@ promBundle.promClient.collectDefaultMetrics();
 // see https://github.com/jochen-schweizer/express-prom-bundle#sample-uusage
 app.use(promBundle.middleware);
 
-app.use(helmet.noCache());
-app.use(helmet.contentSecurityPolicy({
-  directives: {
-    defaultSrc: ['\'self\''],
-    imgSrc: ['\'self\'', 'data:'],
-    styleSrc: ['\'unsafe-inline\''],
+app.use(helmet({
+  noCache: true,
+  frameguard: { action: 'deny' },
+  hsts: { includeSubDomains: false },
+  contentSecurityPolicy: {
+    directives: {
+      defaultSrc: ['\'self\''],
+      imgSrc: ['\'self\'', 'data:'],
+      styleSrc: ['\'unsafe-inline\''],
+    }
   },
```

Comment:
There doesn't seem to be any changes related to making third party domains less specific.